alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -2263,7 +2263,7 @@ public static function getVolumeMapContent($flavorAsset, $mapScale = null, $dura
 		$content = null;
 
 		$cacheStore = kCacheManager::getSingleLayerCache(kCacheManager::CACHE_TYPE_VOLUME_MAP);
-		$cacheKey = $entryId . '_volumeMap';
+		$cacheKey = $entryId . '_volumeMap' . $mapScale;
```

Comment:
$cacheKey = $entryId . '_volumeMap' . $mapScale;
to
$cacheKey = $entryId . '_volumeMap**_**' . $mapScale;